docs/Developers/Design/deploy_procedure_design.md

```diff
@@ -0,0 +1,42 @@
+This document describes the actions that have to be executed successfully for
+monstorak's CRDs to be considered reconciled.
+
+![ActionGraph](workflow.dot.png)
```

Comment:
Diagram needs to be updated. Remove StorageMonitoringPlan.

Reply:
ack